### ISSUE TITLE
[Bug]: Web Widget Zoom Resets to 150% After Refresh

### ISSUE THREAD

**Issue by ShadowArcanist** [wavetermdev/waveterm#1611]
### Current Behavior

Web Widget - After setting the zoom factor to 100% or Reset and refreshing the widget, the content defaults to a zoom level of somewhere around 150%.

### Expected Behavior

The zoom factor should remain at the user-selected level (100% or Reset) after refreshing the widget.

### Steps To Reproduce

1. Open any website on the Web widget
2. Set the web widget zoom factor to 100% or Reset.  
3. Refresh the widget by clicking the refresh icon.  
4. The content inside the widget zooms to around 150% automatically.

### Wave Version

0.10.4 (202412210242)

### Platform

macOS

### OS Version/Distribution

macOS Sonoma 14.7.1 (23H222)

### Architecture

arm64

### Anything else?

https://github.com/user-attachments/assets/c8cdb5d0-c47c-4061-b43d-96500c65c07e

- This issue is not limited to custom web widgets

### Questionnaire

- [ ] I'm interested in fixing this myself but don't know where to start
- [ ] I would like to fix and I have a solution
- [ ] I don't have time to fix this right now, but maybe later

**Comment by esimkowitz** [comment]
Can you do the following for me: open a web widget and a terminal widget next to each other as shown:
<img width="1740" alt="Screenshot 2024-12-23 at 4 17 24 PM" src="https://github.com/user-attachments/assets/201ef00d-fb68-4ace-af84-1fe74bef5dd3" />

Run `wsh getmeta -b 1`, then do the force reload that loses the zoom, then run this again and paste the output as a response to this.

**Comment by esimkowitz** [comment]
Also do you have `web:zoom` defined in your `settings.json`?

**Comment by ShadowArcanist** [comment]
> Can you do the following for me: open a web widget and a terminal widget next to each other as shown: <img alt="Screenshot 2024-12-23 at 4 17 24 PM" width="1740" src="https://private-user-images.githubusercontent.com/16651283/398259057-201ef00d-fb68-4ace-af84-1fe74bef5dd3.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzUzMjY4MTcsIm5iZiI6MTczNTMyNjUxNywicGF0aCI6Ii8xNjY1MTI4My8zOTgyNTkwNTctMjAxZWYwMGQtZmI2OC00YWNlLWFmODQtMWZlNzRiZWY1ZGQzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMjclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjI3VDE5MDgzN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWIxYTM2Y2UyZDhlNThhYjFlYTdiN2U1MjM1OWFiOGU1ZjU4ZjQzMTk5OGQ2Y2E1ZjNiOWNmZWE5YTk2NTgyNjkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.ATXUvpqZYzt2pvUoC0ONzEBaak-ADLq8E2oziebPYX8">
> 
> Run `wsh getmeta -b 1`, then do the force reload that loses the zoom, then run this again and paste the output as a response to this.


Hey @esimkowitz this is what I got
![image](https://github.com/user-attachments/assets/3d8bb23c-0934-48cc-ba1e-944fd3546791)


I don't have anything related to Zoom on settings.json:
![image](https://github.com/user-attachments/assets/e7d4b6dc-4c29-4c2a-b751-6fb558c7019f)


After you asked "what is the zoom level for your window?" I reset my window zoom (mac topbar > view > reset zoom) then I tried changing the zoom level to 100% on the web widget and reloaded the web widget nothing changed on Zoom visually.

I then we to mac topbar > view > zoom in (clicked only one time) and tried changing the zoom level to 100% on the web widget and reloaded the web widget - this time the web widget zoom level changed to 120% after the reload, seems like the web widget it setting it's Zoom level to match the window zoom level

**Comment by esimkowitz** [comment]
Yeah we set the web zoom factor relative to the window zoom factor.

**Comment by ShadowArcanist** [comment]
Hey @esimkowitz I forgot to sent a response! (sorry for the delay)

I started using wave at the default 100% window zoom so this is not a deal breaker for me. 

I think we can close this issue 😀
